InfluxData.h

```diff
@@ -18,11 +18,16 @@ class InfluxData {
     _values = (_values == "") ? (" ") : (_values += ",");
     _values += key + "=" + String(value);
   }
+  void setTime(long int value)
```

Comment:
To avoid confusion, it would be better to specify what's expected here. 

Something like`setTimestamp(long int seconds)`